bitmark-references/bitmark-sdk/websocket.md

```diff
@@ -0,0 +1,85 @@
+# Web Socket
+Realtime event triggering is trending in modern application development nowadays. It helps the application is always up to date and so provides better user experience.
+Bitmark SDK offers Web Socket event triggering to help the application can be notified immediately when there are anything changes.
+
+## Event
+- New block: This event will be sent when bitmark blockchain has a new block.
+  + `blockNumber` : new block number
+- Bitmark changed: This event will be sent when users issue new bitmarks, received new bitmarks, or send them to new owner.
+  + `bitmarkId`: the id of bitmark is changed
+  + `txId`: the corresponding transaction id
+  + `presence`: indicate that the bitmark is owned or not by the corresponding `Account`
```

Comment:
We didn't explain the `Account`. It could be confusing. @anhnguyenbitmark Do you still have your proposal doc for websocket? Probably we can use it for further explanation here.

Reply:
https://github.com/bitmark-inc/core-api/tree/develop/command/event-notifier

Reply:
So the concept of `namespace` and `channel` should be introduced.

Reply:
I don't think we need to introduce the `namespace` and `channel` in SDK doc. It should be mentioned in API doc for their own implementation without using our SDK. In SDK, we just provide specific methods to subscribe to something, not allow to passing `namespace` and `channel`. It would be like `subscribeNewBlock` and should not be `subscribe(Channel)`

Reply:
Understood. We might need to add more examples about how to do the subscriptions?

Reply:
I just added one example and i think it's enough for the instruction. Other ones are similar and we also already introduced the attributes meaning for the corresponding event in above section.